scripts/build/rollup-plugin-javascript-prefixer.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable import/no-extraneous-dependencies */
+const glob = require('glob');
+const fs = require('fs').promises;
+
+const componentPackageJson = require(`${process.cwd()}/package.json`);
+
+const JS_FILES_TO_PREFIX = glob
+  .sync('**/*.js', {
+    ignore: [
+      'node_modules/**/*',
+      'dist/**/*',
+      'lib/**/*',
+      'index.react.js',
+      '**demo.js',
+      '**.bak',
+      '**test**',
+      '**story**',
+    ],
+  })
+  .map(f => `./${f}`);
+
+const _getComponentInfo = () => {
+  const elementMapping = new Map();
+  elementMapping.set('10-atoms', 'a-');
+  elementMapping.set('20-molecules', 'm-');
+  elementMapping.set('30-organisms', 'o-');
+
+  const cwdAsStringArray = process.cwd().split('/');
```

Comment:
@raphaellueckl use OS dependent var `path.sep` for PATH splits:  https://nodejs.org/api/path.html#path_path_sep